test/end-to-end/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "end-to-end",
+  "version": "0.0.0",
+  "description": "weldr-web end-to-end automation test",
+  "scripts": {
+    "test": "mocha -R xunit -O output=result.xml"
+  },
+  "dependencies": {
```

Comment:
Reasons for setup a separate package.json:

1. The docker container for end-to-end test just includes files inside end-to-end directory. So I need a package.json to do environment provision.

2. There're tons of packages need to be installed in project level. That'll spend a lot of time on package installation. But end-to-end test just need some of them. So the separate package.json saves a lot of package installation times.